### PR TITLE
fix(risk): make canonical CII the only product source

### LINKED ISSUE #5057: [P1] Make CII scoring canonical and align score thresholds/methodology
## Summary

Validated audit finding: CII and related scoring surfaces still have multiple live computation paths and threshold drift. This can make different parts of the app show different risk truth depending on cache/auth/fallback state.

### CII dual-engine divergence

Evidence:

- `src/components/CIIPanel.ts` calls local `calculateCII()` in `refresh`.
- `src/app/data-loader.ts` applies cached server scores when present, but falls back to local `calculateCII()` for map scores when cached risk scores are unavailable or force-local is true.
- Server `get-risk-scores` is treated elsewhere as the authoritative source.

Impact:

- Panel/map/server/API can diverge when cached server scores are unavailable.
- Product semantics change silently instead of showing "risk scores unavailable/degraded".

### CII threshold drift

Evidence:

- Strategic composition uses `score >= 50` for elevated-ish countries in `src/services/cross-module-integration.ts` and `src/components/StrategicRiskPanel.ts`.
- Formal CII elevated band is `>= 51`; score 50 is still normal by the formal band.

Impact:

- Strategic risk composition and UI grouping can disagree with formal CII bands.

### Resilience pillar-combine default/env mismatch risk

Evidence:

- `server/worldmonitor/resilience/v1/_shared.ts::isPillarCombineEnabled` defaults to `false` unless `RESILIENCE_PILLAR_COMBINE_ENABLED=true`.
- Audit notes methodology docs claim production is expected to run with this enabled.

Impact:

- Local/sidecar/prod or subsets can compute different overall resilience scores if env is not aligned.

## Recommended Fix Shape

1. Decide the canonical scoring source for CII. Prefer server `get-risk-scores` as the single product truth.
2. Replace silent local fallback with an explicit degraded/unavailable state, or guarantee local scoring uses the exact same versioned inputs/weights as server.
3. Align elevated threshold usage to the formal band (`>= 51`) or explicitly document a different strategic-risk threshold.
4. Add tests that compare CII band boundaries at 50/51 across panel, strategic risk, and server outputs.
5. Verify production env for `RESILIENCE_PILLAR_COMBINE_ENABLED`; update docs/defaults/cache-prefix handling so runtime behavior matches methodology.

## Acceptance Criteria

- CII panel, map, and API use the same canonical scoring semantics.
- If canonical scores are unavailable, users see a degraded/unavailable state rather than silent local divergence.
- Threshold tests cover score 50 and 51.
- Resilience pillar-combine behavior is explicit in docs and tests, and env/default mismatch cannot silently change score methodology.

## Grouping Notes

This issue groups product-truth scoring risks. It may need design/owner input before implementation because it touches methodology, UI fallback, and cache behavior.

### PR DESCRIPTION
**Pull request by koala73**
## Summary

When canonical CII data was unavailable, different product surfaces could silently recompute or reinterpret risk locally, so the same country could show conflicting scores and severity. This change makes cached server CII the only product source: panels, maps, search, country briefs, strategic risk, alerts, and story sharing now use the same canonical levels or show an explicit unavailable state.

It also makes the pillar-combined resilience methodology the production default, with only an explicit normalized `false` restoring the legacy six-domain formula. The formal elevated boundary is now consistently 51 across product consumers.

## Behavior and design decisions

- Cache loss fails closed instead of publishing a locally calculated score; repeated unavailable or unchanged-cache refreshes are deduplicated.
- Country Brief styling, strategic-risk summaries, map/search labels, and cross-module alerts consume canonical `CountryScore.level` semantics.
- Story cards and share text report CII as unavailable without fabricating `0/100`, `NORMAL`, or `STABLE`.
- Persisted cached risk payloads are validated at the boundary before being treated as authoritative.
- Legacy resilience formula tests opt into the rollback flag explicitly, while default-on activation and cache/formula tags are covered separately.

## Validation

- Full data suite: 14,552 passed, 0 failed, 6 intentional skips.
- Focused CII/resilience suite: 130 passed, 0 failed.
- Frontend and API typechecks passed.
- Lint and safe-HTML guard passed (existing repository warnings only).
- Full production Vite build passed.
- Browser smoke test with canonical scores unavailable showed explicit `UNAVAILABLE` states for Country Instability and Strategic Risk, with no local score rendered.
- Pre-push gate passed, including architectural boundaries, edge bundling, resilience tests, docs/MDX checks, and version sync.

## Post-Deploy Monitoring & Validation

- Watch risk-score RPC availability, stale/degraded cache rates, and client logs for `Canonical backend risk scores unavailable` or unexpected CII unavailable-state volume.
- Confirm panel, map, search, country brief, strategic-risk, and story surfaces agree on sampled country scores and the 50/51 boundary after deployment.
- Confirm resilience responses and cache keys use the pillar-combined formula tag; investigate any legacy-tag cache served without an explicit rollback flag.
- Healthy state: canonical scores render consistently, unavailable states recover after cache restoration, and no product surface reports a local calculation.
- Roll back pillar combination with `RESILIENCE_PILLAR_COMBINE_ENABLED=false` if published rankings show a methodology/cache mismatch; a widespread canonical-score outage should be handled by restoring the backend/cache rather than re-enabling browser fallbacks.
- Monitor through the first deployment and for 24 hours; release owner owns rollback and cache invalidation if the failure signals appear.

Fixes #5057.

---

[Compound Engineering](https://github.com/EveryInc/compound-engineering-plugin)
Codex

### CODE REVIEW

File: src/app/data-loader.ts [hunk 1]
```diff
@@ -512,45 +511,43 @@ export class DataLoaderManager implements AppModule {
...
+  private appliedCiiState: CachedRiskScores | null | undefined;
+
   private applyCiiScoresToMap(scores: CountryScore[]): void {
     this.ctx.map?.setCIIScores(scores.map(s => ({ code: s.code, score: s.score, level: s.level })));
     this.ctx.map?.setLayerReady('ciiChoropleth', scores.length > 0);
   }
 
-  private renderCachedCiiScores(cached: CachedRiskScores): void {
+  private renderCachedCiiScores(cached: CachedRiskScores): boolean {
     this.cachedRiskScores = cached;
```
[the remark this answers is not in the document]
Comment (koala73) [reply src/app/data-loader.ts]: Addressed in 222dbbc31. I removed DataLoaderManager's private memo entirely rather than retaining a stale fallback: every refresh now reads getCachedScores(), while appliedCiiState only deduplicates panel and map rendering. The regression covers the first cache, the same reference, a fresh cache object, and cache loss back to unavailable.

File: server/worldmonitor/resilience/v1/_shared.ts [hunk 2]
```diff
@@ -67,7 +66,7 @@ export const RESILIENCE_SCHEMA_V2_ENABLED =
 // prefix or waiting for the 6h TTL to expire — otherwise legacy
 // 6-domain scores will be served from cache after activation.
 export function isPillarCombineEnabled(): boolean {
-  return (process.env.RESILIENCE_PILLAR_COMBINE_ENABLED ?? 'false').toLowerCase() === 'true';
+  return process.env.RESILIENCE_PILLAR_COMBINE_ENABLED?.trim().toLowerCase() !== 'false';
 }
```
[the remark this answers is not in the document]
Comment (koala73) [reply server/worldmonitor/resilience/v1/_shared.ts:70]: Addressed in 222dbbc31. The comment now documents that only normalized false disables pillar combine, while 0, no, and off remain enabled. It also records that the payload formula tag blocks cross-formula cache reuse and that prefix bumps are for scorer changes within the same formula tag.
